Models/Crime.cs

```diff
@@ -1,36 +1,24 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-// <auto-generated />
-//
-// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
-//
-//    using QuickType;
-//
```

Comment:
Good to remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.